src/ic-websocket-cdk/src/lib.rs

```diff
@@ -30,8 +30,12 @@ const LABEL_WEBSOCKET: &[u8] = b"websocket";
 const DEFAULT_MAX_NUMBER_OF_RETURNED_MESSAGES: usize = 50;
 /// The default interval at which to send acknowledgements to the client.
 const DEFAULT_SEND_ACK_INTERVAL_MS: u64 = 300_000; // 5 minutes
+/// The maximum latency allowed between the client and the canister.
+const MAX_ALLOWED_CONNECTION_LATENCY_MS: u64 = 30_000; // 30 seconds
```

Comment:
I'd call it `MAX_ALLOWED_COMMUNICATION_DELAY_MS`

Reply:
Yeah I was also thinking that this is not the best name